react-sdk/src/hooks/use-tambo-thread-messages.ts

```diff
@@ -0,0 +1,15 @@
+import { useTambo } from "../providers/tambo-provider";
+
+/**
+ * Hook to access the current thread's messages and generation state.
+ * @returns Object containing messages array, generation status, and current stage
+ */
+export function useTamboThreadMessages() {
```

Comment:
I don't see what this buys us? we have useTambo() which gives direct access to this stuff, and we're just introducing "isGenerating" as the inverse of "isIdle" which is going to make the API really confusing without reading the source ("can isIdle and isGenerating both be true? which one is authoritative?)